types/config.go

```diff
@@ -206,6 +209,12 @@ type RunConfig struct {
 	// InstanceGroup
 	InstanceGroup string
 
+	// AwsIPN is Aws instance profile name
```

Comment:
we'd like these to be cloud agnostic as much as possible so should drop the 'aws' prefix